src/main/scala/com/cloudera/finance/parsers/CSVParser.scala

```diff
@@ -13,31 +13,33 @@
  * License.
  */
 
-package com.cloudera.finance
+package com.cloudera.finance.parsers
 
 import com.cloudera.sparkts.TimeSeries
 import com.cloudera.sparkts.TimeSeries._
 
-import org.apache.spark.SparkContext
-import org.apache.spark.rdd.RDD
-
-import org.joda.time.DateTime
-
-object YahooParser {
-  def yahooStringToTimeSeries(text: String, keyPrefix: String = ""): TimeSeries = {
-    val lines = text.split('\n')
+trait CSVParser extends Parser {
```

Comment:
So the assumption here is that the first column in the CSV is a timestamp and then we create a column for the TimeSeries out of every other column?  If so, the name `CSVParser` seems maybe too general.  Perhaps call it TimestampedCSVParser instead and add some header doc explaining what it does.